pkgs/os-specific/linux/pam_ssh_agent_auth/default.nix

```diff
@@ -1,31 +1,22 @@
-{ stdenv, fetchpatch, fetchurl, pam, openssl, perl }:
+{ stdenv, fetchpatch, fetchgit, pam, openssl, perl }:
 
 stdenv.mkDerivation rec {
-  name = "pam_ssh_agent_auth-0.10.3";
+  name = "pam_ssh_agent_auth-git-master";
```

Comment:
Strings like `git-master` should be avoided since they're not a version (they don't start with a digit). E.g. `nix-env` won't consider this the same package anymore.

Reply:
The problem is that they didn't update the version, so their version number is lower than the original but they have more fixes :-( should I just use their version?

Reply:
Probably.  I don't think this will be commonly installed by `nix-env`, and on other places you don't care about version "downgrade" AFAIK.

Reply:
You can use a version like `20200612` or whatever the date of git master is.

Reply:
Just a little correction, that the manual says to do like `2020-06-12` with dashes